packages/netlify-cms-backend-github/src/implementation.js

```diff
@@ -142,7 +143,7 @@ export default class GitHub {
 
     await getPermissionToFork();
 
-    const fork = await fetch(`${this.api_root}/repos/${this.originRepo}/forks`, {
+    const fork = await fetch(`${this.api_root}${this.repoURLPrefix}${this.originRepo}/forks`, {
```

Comment:
And here as well.

Reply:
Yep so `this.options.proxied` will be false and this will still evaluate `/repos/` - I don't think this should be a problem? Is there any benefit to having `/repos/` hardcoded in the method?

Reply:
It's just for readability and debugging sake. With the `this.options.proxied` check, if I  am looking at the code, I am thinking the `git-gateway` is calling those APIs.